packages/checkout/src/checkout.ts

```diff
@@ -87,6 +90,22 @@ export default class Checkout extends EventEmitter2 {
     });
   }
 
+  resize(event: string): void {
```

Comment:
We have a library which we use, which you should be able to instruct to handle this for you. It takes care of more corner cases and less straightforward scenarios. 

I don't really know how exactly it is used, but the name is

`iframe-resizer`. It is loaded into both the host page and the iframe itself. Perhaps Ash or Kev can help you, as I think they worked with this.